### PR TITLE
Fixed issue of header missing in susie combined cred file in rare

### PR DESCRIPTION
**Pull request by Fedja**
occasions when susie returns empty result in a region and it happens to
be the first one in combin stage.

Added wdl to post-hoc filter and summarize full finemap run

Fixed duplicate cs column in filter and summarize

### CODE REVIEW

File: wdl/finemap_sub.wdl [hunk 1]
```diff
@@ -531,14 +534,18 @@ task combine {
         }
         ' ${sep=" " susie_cred} | bgzip -c -@ ${cpu} > ${pheno}.SUSIE.cred.bgz
 
+        FNR == 1 { gsub(/[ \t]+$/, "", $0); if(header_printed==0 && $0!=""){ print "trait", "region", $0header_printed=1; } }
```
Comment (mkanai) [inline wdl/finemap_sub.wdl:537]: You have unnecessary line here.

**Review by mkanai — CHANGES_REQUESTED** [review]
Thanks Mitja! It looks great except a single line which seems some typo to me.

File: wdl/finemap_sub.wdl (hunk 1 above)
Comment (Fedja) [reply wdl/finemap_sub.wdl]: agh... removed

Comment (mkanai) [reply wdl/finemap_sub.wdl]: thanks!

**Review by mkanai — APPROVED** [review]
Looks great now!

**Pull request merged**